http/src/main/resources/init-resources/jupyter-docker-compose-gce.yaml

```diff
@@ -28,6 +28,7 @@ services:
       PIP_USER: "false"
       PIP_TARGET: "${NOTEBOOKS_DIR}/packages"
       R_LIBS: "${NOTEBOOKS_DIR}/packages"
+      PYTHONPATH: "${PYTHONPATH:+$PYTHONPATH:}${NOTEBOOKS_DIR}/packages"
```

Comment:
Trying to understand this syntax.. does this result in a `PYTHONPATH` of `/etc/jupyter/custom:/home/jupyter-user/notebooks` in the container?

Reply:
yeah, i added this in this PR so we can account for users that use old docker images that don't have the `/packages` dir in their `PYTHONPATH`

Reply:
Yeah makes sense to add it here, was just confused about the bash syntax

Reply:
actually removed this line and punting on it for now. it was breaking gce vm creation so need to figure out how to do it correctly but for now going to get the original fix in